packages/block-library/src/template-part/edit/notice.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+
+export default function TemplatePartNotice( { postId, slug } ) {
```

Comment:
I feel like "notice" might be the wrong name for this -- I almost think "label" is better. But names are hard, so not sure :)

Reply:
I totally agree. I was about to push a rebase now with some updates 🙂